doc/api/crypto.md

```diff
@@ -10,6 +10,8 @@ wrappers for OpenSSL's hash, HMAC, cipher, decipher, sign, and verify functions.
 Use `require('crypto')` to access this module.
 
 ```js
+// RUNKIT-ENABLE
```

Comment:
I prefer the way https://github.com/nodejs/node/pull/22831 marked runkit after the "```js" fence rather than inside it like it is here. This comment is visible when rendered as markdown (and maybe in the json version of the docs too?).

Reply:
it wouldn't highlight correctly in any editors i tried with that approach.

Reply:
Should we maybe land this as is for now and try to improve the solution later on?
It seems like it would even be an improvement having the comment in markdown compared to now.

Reply:
I don't understand why this comment line is being added into only one of the js examples, is it the intention that it later be added to every single one? That seems overly intrusive.